### ISSUE TITLE
lambdalabs RTX 2080 Ti Deep Learning Benchmarks

### ISSUE THREAD

**Issue by l1t1** [gcp/leela-zero#1898]
https://lambdalabs.com/blog/2080-ti-deep-learning-benchmarks/
https://github.com/lambdal/lambda-tensorflow-benchmark
TL;DR
•The RTX 2080 Ti’s single-precision (FP32) training of CNNs with TensorFlow is between 27% and 45% faster than the 1080 Ti for measured networks.
•The RTX 2080 Ti’s half-precision (FP16) training of CNNs with TensorFlow is between 60% and 65% faster than the 1080 Ti for measured networks.
•If you do FP16 training, the RTX 2080 Ti is probably worth the extra money. If you don't, then you'll need to consider whether a 71% increase in cost is worth an average of 36% increase in performance.

The 2080 Ti offers a 1.41x speedup over the 1080 Ti for single precision ResNet-152 training.

The 2080 Ti offers a 1.65x speedup over the 1080 Ti for half precision ResNet-152 training.

**Comment by pondturtle** [comment]
If I understood correctly. Right now LZ is not able to take advantage of tensor units on the chip right?

**Comment by Yakago** [comment]
How do these results relate to the performance wrt. inference?

**Comment by AncalagonX** [comment]
Unfortunately, the linked benchmark is only for those who use tools like Tensorflow, etc. for training neural networks.

Nvidia has claimed the Tensor Cores also speed up "inference" (i.e. playing against LZ weights with leelaz.exe, Lizzie, Sabaki, etc.) but I haven't seen any hard details to date on what kind of special performance increases to expect aside from the card itself being more powerful across the board.

I'd like to see some benchmarks from users who own these cards using LZ.

**Comment by gcp** [comment]
>Nvidia has claimed the Tensor Cores will speed up "inference" as well 

They probably mean: if you are using their library (cuDNN), which has a very problematic license for us.

Most of the gain on those cards will come from their fp16 support. There's a few people who wrote cuDNN backends and were disappointed by the performance. Maybe with Tensor Cores there might be a point to using it.

**Comment by lonemonkeywithwhiteshell** [comment]
So,can you expect how much speed up does 2080ti offer over the 1080ti for Leela?

**Comment by lonemonkeywithwhiteshell** [comment]
I recently got a pair of 2080ti. I'm currently figuring out how I can port the cudnn-fp16 implementation (with CUBLAS_

Great!
